monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -0,0 +1,248 @@
+                    xform["pixdim"] = deepcopy(spacing)
+                elif isinstance(xform, str) and xform.startswith("PLACEHOLDER_INTENSITY_NORMALIZATION"):
+                    if modality.startswith("ct"):
+                        self.cfg[f"{key}#transforms#{idx}"] = deepcopy(ct_intensity_xform)
+                    else:
+                        self.cfg[f"{key}#transforms#{idx}"] = deepcopy(mr_intensity_transform)
+        # self.algorithm_dir = os.path.join(self.output_path, "segresnet2d")
+        # self.inference_script = "scripts/infer.py"
+
+
+class DintsAlgo(BundleAlgo):
+    def fill_template_config(self, data_stats_filename):
+        print("implement dints template filling method")
+
+
+default_algos = {
+    "segresnet2d": dict(
+        _target_="SegresnetAlgo",
+        template_configs="../algorithms/templates/segresnet2d/configs",
+        scripts_path="../algorithms/templates/segresnet2d/scripts",
+    ),
+    "dints": dict(
+        _target_="DintsAlgo",
+        template_configs="../algorithms/templates/dints/configs",
+        scripts_path="../algorithms/templates/dints/scripts",
+    ),
+}
+
+
+class BundleGen(AlgoGen):
+    """
+    This class generates a set of bundles according to the cross-validation folds, each of them can run independently.
+
+    .. code-block:: bash
+
+        python -m monai.apps.auto3dseg BundleGen generate --data_stats_filename="../algorithms/data_stats.yaml"
+
+    """
+
+    def __init__(self, algos=None, data_stats_filename=None, data_lists_filename=None):
+        self.algos = []
+        if algos is None:
+            for algo_name, algo_params in default_algos.items():
+                self.algos.append(ConfigParser(algo_params).get_parsed_content())
+                self.algos[-1].name = algo_name
+        else:
+            self.algos = ensure_tuple(algos)
+
+        self.data_stats_filename = data_stats_filename
+        self.data_lists_filename = data_lists_filename
+        self.history = []
+
+    def set_data_stats(self, data_stats_filename):
+        self.data_stats_filename = data_stats_filename
+
+    def get_data_stats(self, fold_idx=0):
+        return self.data_stats_filename
+
+    def set_data_lists(self, data_lists_filename):
+        self.data_lists_filename = data_lists_filename
+
+    def get_data_lists(self, fold_idx=0):
+        return self.data_lists_filename
+
+    def get_history(self, *args, **kwargs):
+        return self.history
+
+    def generate(self, fold_idx: Union[int, Sequence[int]] = 0):
+        for algo in self.algos:
+            for f_id in ensure_tuple(fold_idx):
+                data_stats = self.get_data_stats(fold_idx)
+                data_lists = self.get_data_lists(fold_idx)
+                algo.set_data_stats(data_stats)
+                algo.set_data_source(data_lists)
+                name = f"{algo.name}_{f_id}"
+                algo.export_to_disk(".", algo_name=name)
```

Comment:
The folder location of "." is suboptimal when I was trying to build unit test and save the files to a temporary directory. Can it be a user-specified optional argument? 

Thanks